### PR TITLE
[Data Prepper] MAINT: document on disable_refresh secret extension setting

### PR DESCRIPTION
**Pull request by chenqi0805**
### Description
_Describe what this change achieves._

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dlvenable** [comment]
@chenqi0805 , I changed the target to `data-prepper-2.7` since `disable_refresh` is coming in 2.7.

**Review by dlvenable — APPROVED** [review]
Thanks @chenqi0805 !

File: _data-prepper/managing-data-prepper/configuring-data-prepper.md [hunk 1]
```diff
@@ -148,7 +149,8 @@ Option | Required | Type | Description
...
 region | No | String   | The AWS region of the secret. Defaults to `us-east-1`.                                                                                                                                                                            
 sts_role_arn | No | String   | The AWS Security Token Service (AWS STS) role to assume for requests to the AWS Secrets Manager. Defaults to `null`, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html). 
-refresh_interval | No | Duration | The refreshment interval for AWS secrets extension plugin to poll new secret values. Defaults to `PT1H`. See [Automatically refreshing secrets](#automatically-refreshing-secrets) for details.                             
+refresh_interval | No | Duration | The refreshment interval for AWS secrets extension plugin to poll new secret values. Defaults to `PT1H`. See [Automatically refreshing secrets](#automatically-refreshing-secrets) for details.
```
Comment (Naarcha-AWS) [inline _data-prepper/managing-data-prepper/configuring-data-prepper.md:152]:
```suggestion
refresh_interval | No | Duration | The refreshment interval for AWS secrets extension plugin to poll new secret values. Defaults to `PT1H`. For more information. see [Automatically refreshing secrets](#automatically-refreshing-secrets) for details.
```

File: _data-prepper/managing-data-prepper/configuring-data-prepper.md [hunk 2]
```diff
@@ -148,7 +149,8 @@ Option | Required | Type | Description
...
+disable_refresh | No | Boolean | The flag to disable AWS secrets extension plugin regular polling on latest secret values. Defaults to `false`. When set to `true`, `refresh_interval` will not be used.
```
Comment (Naarcha-AWS) [inline _data-prepper/managing-data-prepper/configuring-data-prepper.md:153]:
```suggestion
disable_refresh | No | Boolean | Disables regular polling on the latest secret values inside the AWS secrets extension plugin. Defaults to `false`. When set to `true`, `refresh_interval` will not be used.
```
```suggestion
disable_refresh | No | Boolean | The flag to disable AWS Secrets extension plugin regular polling on latest secret values. Defaults to `false`. When set to `true`, the `refresh_interval` will not be used.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]

File: _data-prepper/managing-data-prepper/configuring-data-prepper.md [hunk 3]
```diff
@@ -148,7 +149,8 @@ Option | Required | Type | Description
...
 region | No | String   | The AWS region of the secret. Defaults to `us-east-1`.                                                                                                                                                                            
 sts_role_arn | No | String   | The AWS Security Token Service (AWS STS) role to assume for requests to the AWS Secrets Manager. Defaults to `null`, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html). 
-refresh_interval | No | Duration | The refreshment interval for AWS secrets extension plugin to poll new secret values. Defaults to `PT1H`. See [Automatically refreshing secrets](#automatically-refreshing-secrets) for details.                             
+refresh_interval | No | Duration | The refreshment interval for the AWS Secrets extension plugin to poll new secret values. Defaults to `PT1H`. For more information. see [Automatically refreshing secrets](#automatically-refreshing-secrets) for details.
```
Comment (Naarcha-AWS) [inline _data-prepper/managing-data-prepper/configuring-data-prepper.md:152]:
```suggestion
refresh_interval | No | Duration | The refreshment interval for the AWS Secrets extension plugin to poll new secret values. Defaults to `PT1H`. For more information, see [Automatically refreshing secrets](#automatically-refreshing-secrets).
```

**Review by Naarcha-AWS — APPROVED** [review]

**Comment by dlvenable** [comment]
@Naarcha-AWS , Are we good to merge this?

**Pull request merged**